docs/snippets/ov_extensions.cpp

```diff
@@ -94,14 +94,14 @@ core.add_extension(ov::frontend::OpExtension<CustomOperation>());
 
 //! [frontend_extension_CustomOperation_rename]
 core.add_extension(ov::frontend::OpExtension<CustomOperation>(
-    { {"attr1", "fw_attr1"}, {"attr2", "fw_attr2"} },
+    std::map<std::string, std::string>{ {"attr1", "fw_attr1"}, {"attr2", "fw_attr2"} },
```

Comment:
Why do you need this change?

Reply:
The OpExtensionBase added new Constructor which makes some compiler may fail to do the template instantiation. To avoid the errors here need to give the type information to not worked compiler.

Reply:
New constructor can be explicit in order to avoid changes in released snippets

Reply:
Adding explicit could not resolve the problem, it can just [prevent implicit conversion](https://stackoverflow.com/questions/121162/what-does-the-explicit-keyword-mean) that it's not our case. I add a tag parameter to the new constructor in order to make an explicit difference between new and old ones.